client/src/components/profile/AuthenticationCard.vue

```diff
@@ -114,6 +114,13 @@ const methodConfig: Record<
     methodName: '',
     description: '',
   },
+  // TODO
+  [DeviceSaveStrategyTag.OpenBao]: {
```

Comment:
It should replace the `CustomDeviceSaveStrategyTag.SSO` just below.

Reply:
I guess the `imageAlt: 'Smartcard',` should be changed, and also `sso` should be renamed to `openbao` for consistency.
So what about doing this is another PR ? ^^